structs.h

```diff
@@ -1387,7 +1387,7 @@ typedef struct PkPlayerStruct {
 	PkItemStruct InvBody[7];
 	PkItemStruct InvList[40];
 	char InvGrid[40];
-	char _pNumInv;
+	BYTE _pNumInv;

```

Comment:
I wonder what do the `p` prefixes stand for in this struct?  At first glance I thought about pointers.  Now I got lost.   Maybe "player" ?

Reply:
Yeah, they abbreviated most things. `_i` for item, `_m` for monster, etc..